vis.c

```diff
@@ -294,7 +296,10 @@ static void window_draw_colorcolumn(Win *win) {
 
 		/* This screen line contains the cell we want to highlight */
 		if (cc <= line_cols + width) {
-			l->cells[(cc - 1) - line_cols].style = style;
+			CellStyle* orig = & l->cells[(cc - 1) - line_cols].style;
```

Comment:
The placement of `*` and `&` isn't really inline with the rest of the code base. Also while you are changing that code section, the parentheses are unnecessary:

```
CellStyle *orig = &l->cells[cc - 1 - line_cols].style;
```

Furthermore, there is no need for two separate commits (also applies to #818). If you squash and rebase, I will merge it.

Thanks